### ISSUE TITLE
Project is configured and running but Localhost is not able to load or Keycloak url is not working  ?

### ISSUE THREAD

**Issue by Abhineet786** [AOT-Technologies/forms-flow-ai#1408]
Project is configured here...

<img width="1329" alt="Screenshot 2023-04-24 at 2 52 21 PM" src="https://user-images.githubusercontent.com/131658991/233955177-4c5fcc57-a4c1-49f6-a9c9-9ae478239ec2.png">

Trying to access localhost here..but no luck .

<img width="1333" alt="Screenshot 2023-04-24 at 2 53 28 PM" src="https://user-images.githubusercontent.com/131658991/233955503-ed845a6a-016d-4217-b6af-27c2bc7bd93e.png">

<img width="1298" alt="Screenshot 2023-04-24 at 2 53 44 PM" src="https://user-images.githubusercontent.com/131658991/233955557-9b174941-4159-4931-8baa-1171d3fa626e.png">

**Comment by Abhineet786** [comment]
i have followed this [https://aot-technologies.github.io/forms-flow-installation-doc/Pages/Server/containerized_deployment.html](url)

**Comment by sumesh-aot** [comment]
Looks like keycloak is not up, can you check if you have anything running on 8080 ?
Also easy way to try out formsflow.ai is using quick installation script : https://aot-technologies.github.io/forms-flow-installation-doc/Pages/PC/PC.html

**Comment by Abhineet786** [comment]
Thanks for the reply [sumesh-aot](https://github.com/sumesh-aot)

i have tried this but getting some issues in Mac M1. I have attached related screenshots below.


<img width="1153" alt="Screenshot 2023-04-25 at 3 17 40 PM" src="https://user-images.githubusercontent.com/131658991/234239836-8a7bb3db-0071-4c22-a1f5-9b424c75083e.png">

<img width="1367" alt="Screenshot 2023-04-25 at 3 15 04 PM" src="https://user-images.githubusercontent.com/131658991/234239107-7ed3335f-def6-4a54-9cce-9f2f71809c9c.png">

**Comment by Abhineet786** [comment]
How much Containerized deployment is compatible with Apple M1 Chip ?
I had following the below documents to configure.
https://aot-technologies.github.io/forms-flow-installation-doc/Pages/Server/containerized_deployment.html

**Comment by sumesh-aot** [comment]
@Abhineet786 are you using the docker-compose file for arm64 ? If not, please try it and update the result here.

**Comment by Abhineet786** [comment]
@sumesh-aot   yes, i am using this file  docker-compose-arm64.yml for docker-compose.

**Comment by sumesh-aot** [comment]
@Abhineet786 Can you try the docker-compose from here https://github.com/AOT-Technologies/forms-flow-ai-deployment/blob/refactor/docker-compose/docker-compose-arm64.yml or try quick install after downloading the repo from `refactor` branch (https://github.com/AOT-Technologies/forms-flow-ai-deployment/tree/refactor). Seems like issue is with keycloak db volume. PR is in progress (https://github.com/AOT-Technologies/forms-flow-ai-deployment/pull/43)

**Comment by Abhineet786** [comment]
Hi @sumesh-aot Trying to run the above repo but getting same issues : 

<img width="1334" alt="Screenshot 2023-04-26 at 4 59 27 PM" src="https://user-images.githubusercontent.com/131658991/234561708-c2a06e7e-86cd-41b2-889f-56e040afc6a2.png">

<img width="1160" alt="Screenshot 2023-04-26 at 4 59 44 PM" src="https://user-images.githubusercontent.com/131658991/234561755-a0069394-db44-4960-8140-bd93661325e1.png">

When i try to access localhost:3001 .... getting un-authorized screen
<img width="1431" alt="Screenshot 2023-04-26 at 5 00 08 PM" src="https://user-images.githubusercontent.com/131658991/234561836-4a98654a-a7c4-442d-9e6e-288e09ce702e.png">

Is this application compatible with 8 Gb RAM ?

**Comment by sumesh-aot** [comment]
@Abhineet786 It should work with 8 GB RAM. Seems like your images are different from the screenshot. The ARM64, docker compose is using different image for keycloak (https://github.com/AOT-Technologies/forms-flow-ai-deployment/blob/refactor/docker-compose/docker-compose-arm64.yml#L17), but your screenshot is saying it's using image from quay. Could you please re-check ?

**Comment by Abhineet786** [comment]
Hi @sumesh-aot  i have also try to setup this (https://aot-technologies.github.io/forms-flow-installation-doc/Pages/Server/containerized_deployment.html) on linux server. But when i am accessing the react or keyclock url with server ip ...page is not loading.

<img width="1438" alt="Screenshot 2023-04-27 at 9 57 51 PM" src="https://user-images.githubusercontent.com/131658991/234928403-495cb103-db67-4cbf-b851-b8a72a9f14e8.png">

took too long to respond.
<img width="1426" alt="Screenshot 2023-04-27 at 10 02 12 PM" src="https://user-images.githubusercontent.com/131658991/234929468-594a8adf-2dd8-435f-a71d-8189f4735829.png">

and one more question should i need to install keycloak package separately  in the system to generate KEYCLOAK_BPM_CLIENT_SECRET or i can use forms-flow-keycloak  http://{ip address of system}:8080 url to generate client secret.

**Comment by sumesh-aot** [comment]
@Abhineet786 You dont need to install keycloak separately as docker-compose-arm64 contains keycloak image.
Can you do a cleanup and install docker-compose-arm64.yml in Mac M1 and update the result. From your previous screenshot it doesn't look like you were using docker-compose-arm64 and instead was using docker-compose.yml.

**Comment by sumesh-aot** [comment]
If you are still facing issues, our team would be able to help troubleshoot the issue over a call.

**Comment by Abhineet786** [comment]
@sumesh-aot thanks for your support, yes i am facing some issues in both mac and linux server. If you can assign someone to troubleshoot these issues, it will be great.

**Comment by Abhineet786** [comment]
@sumesh-aot i am waiting for your response.

**Comment by sumesh-aot** [comment]
@Abhineet786 Sincere apologies, I was on vacation. If you are still facing issues, please share your email.
